### PR TITLE
Fix documentation & coding style of User.name & User.full_name

### PR DESCRIPTION
**Pull request by tsnoam**

### CODE REVIEW

**Comment by bomjacob** [comment]
Maybe note in docs that the username returned starts with '@'?

**Comment by tsnoam** [comment]
@bomjacob Good idea. On it.

**Comment by Eldinnie** [comment]
@tsnoam I would remove the note.

File: telegram/user.py [hunk 1]
```diff
@@ -72,26 +72,27 @@ def __init__(self,
     @property
     def name(self):
         """
-        :obj:`str`: The users :attr:`username` if available, if not it returns the first name and
-            if present :attr:`first_name` and :attr:`last_name`.
+        :obj:`str`: The user's :attr:`username` if available, otherwise returns :attr:`full_name`.
+
+        NOTE: This is a convenience property, not part of the official Telegram Bot API.
```
Comment (Eldinnie) [inline telegram/user.py]: Remove these notes please. We have them nowhere in the docs and it has no added value imo.

File: telegram/user.py [hunk 2]
```diff
@@ -72,26 +72,27 @@ def __init__(self,
...
+            return '@{}'.format(self.username)
+        return self.full_name
 
     @property
     def full_name(self):
         """
-        :obj:`str`: The users :attr:`first_name` and if present :attr:`last_name`.
+        :obj:`str`: The user's :attr:`first_name`, followed by (if present) :attr:`last_name`.
+
+        NOTE: This is a convenience property, not part of the official Telegram Bot API.
```
Comment (Eldinnie) [inline telegram/user.py]: Remove these notes please. We have them nowhere in the docs and it has no added value imo.

**Comment by bomjacob** [comment]
LGTM :)

**Pull request merged**